components/CommonComponents/ShellBox/index.stories.tsx

```diff
@@ -0,0 +1,12 @@
+import ShellBox from '.';
+
+export default {
+  title: 'CommonComponents/ShellBox',
+  component: ShellBox,
+};
+
+export const Default = () => (
+  <ShellBox textToCopy="echo 'Hello World'">
+    echo &rsquo;Hello World&rsquo;
```

Comment:
Since the content will come from a md file, can we use a template string variable to mimic that through props instead of directly writing in the story. Also makes easier to try component in the storybook

Reply:
No it is used to make the shellbox on the index page. It has nothing to do with markdown.